start.js

```diff
@@ -4,12 +4,14 @@ const { dependencies } = require('./package');
 const bodyParser = require('body-parser');
 const apihead = require('./headers/API');
 
+const PORT = process.env.PORT || 90;
+
 // Giving the launch data and start listening
-app.listen(90, function() {
+app.listen(PORT, function() {
     // I know that this can all go into one log but I prefer to log per line to keep it clean
     console.log("\x1b[34mApp started\x1b[0m");
-    console.log("\x1b[33mModules loaded:\n" + JSON.stringify(dependencies, null, 2) + "\x1b[0m")
-    console.log("\x1b[32mserver address: http://localhost:90\x1b[0m");
+    console.log(`\x1b[33mModules loaded:\n${JSON.stringify(dependencies, null, 2)}\x1b[0m`)

```

Comment:
I personally don't like this method of adding variables to a string

Reply:
Why not? I find template literals a lot cleaner to read. (Of course, it's still your call to make.)